modules/ROOT/pages/deploy-to-runtime-fabric.adoc

```diff
@@ -129,6 +130,15 @@ If there is an insufficient number of worker nodes or resources available on the
 +
 If needed, review xref:deploy-resource-allocation.adoc[determine resource allocation] for applications deployed to Runtime Fabric.
 
+=== (Optional) Enable TLS v1 and TLS v1.1
+
+The Mule runtime disables TLS v1 and TLS v1.1 by default. When you deploy an application to Runtime Fabric, you can enable TLS v1 or TLS v1.1 using JVM args.
+
+. Click the *JVM* tab. 
+. In the *Java Virtual Machine arguments* field, add one of the following arguments:
```

Comment:
Do we need to separately mention that these properties can also be passed using other supported deployment methods like API & Mule maven plugin?

Reply:
not here. I fill figure out where to add them in the other places--perhaps in the release notes